src/Protocol/DocumentUri.cs

```diff
@@ -14,17 +15,23 @@ namespace OmniSharp.Extensions.LanguageServer.Protocol
     /// <remarks>This exists because of some non-standard serialization in vscode around uris and .NET's behavior when deserializing those uris</remarks>
     public class DocumentUri : IEquatable<DocumentUri>
     {
+        public static Regex WindowsPath =
+            new Regex("^\\w(?:\\:|%3a)[\\\\|\\/]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+        public string _delimiter = SchemeDelimiter;
```

Comment:
Should this be private? Because of the underscore

Reply:
probably... it was late at night